### PR TITLE
fixing rustdoc warnings

### PR DESCRIPTION
**Pull request by franziskuskiefer**

### CODE REVIEW

**Review by raphaelrobert — APPROVED** [review]
Thanks, LGTM! (just one nit)

**Comment by franziskuskiefer** [comment]
> just one nit

do you have that anywhere @raphaelrobert ? 🙂

File: openmls/src/ciphersuite/mod.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ mod signature;
 
 pub(crate) use aead::*;
 pub use ciphersuites::*;
-pub(crate) use errors::*;
+pub use errors::*;
```
Comment (raphaelrobert) [inline openmls/src/ciphersuite/mod.rs:38]: This will be removed in #592 anyway.

**Review by raphaelrobert — APPROVED** [review]
Heh, looks like GH swallowed my comment from before.

**Comment by franziskuskiefer** [comment]
Ok, I'll leave the change than as is when it's going to be removed in a follow-up anyway.

**Pull request merged**